templates/app/app.hbs

```diff
@@ -33,6 +27,13 @@ const corsConfig = {
   credentials: true,
 };
 
+app.use(morgan('tiny'));
+app.use(cors(corsConfig));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, 'public')));
+
 app.use('/forest/authentication', cors({
```

Comment:
Unfortunately, this route must be declared before:
`app.use(cors(corsConfig))`

Reply:
Thank you. I will update, the only thing we need is to have cors before body-parser :)

Reply:
Any hint why @arnaudbesnier ? On the server, `app.use(cookieParser());` is after `app.use(dynamicCors);`.

Reply:
@ghusse is the best person to explain why:
`app.use('/forest/authentication', cors({...});`
is before
`app.use(cors(corsConfig));`

See the recent contribution about it:
https://github.com/ForestAdmin/lumber/pull/484

I don't get what you have in mind about your last sentence:
> On the server, app.use(cookieParser()); is after app.use(dynamicCors);

Reply:
maybe set @ghusse a reviewer of this?

Reply:
In my PR, `app.use('/forest/authentication', cors({...});` was before `app.use(cors(corsConfig));` but after bodyparser (it was the same as before my modification).

I don't know why the order changed in the meantime.

Anyway, I don't see any problem with calling `cors` before `bodyParser`. The code inside `cors` only needs to access to headers.